packages/nimble-components/src/wafer-map/tests/matrixRenderer.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { WaferMap } from '..';
+import { MatrixRenderer } from '../modules/matrixRenderer';
+
+describe('MatrixRenderer', () => {
+    it('worker should be healthy', () => {
+        const waferMap = new WaferMap();
```

Comment:
I think you can use a WaferMapMock here, please see other tests

Reply:
I deleted this file as we discussed, I think this comment is not longer needed